app/src/main/res/layout/item_album_detail.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <data>
+
+        <variable
+            name="track"
+            type="com.framgia.soundclound.data.model.Track"/>
+        <variable
+            name="listener"
+            type="com.framgia.soundclound.screen.BaseOnItemClick"/>
+    </data>
+
+    <android.support.constraint.ConstraintLayout
+        android:background="@color/color_orange"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/dp_72" >
+
+        <ImageView
+            android:contentDescription="@string/image_track"
+            android:id="@+id/imv_track"
+            app:layout_constraintRight_toLeftOf="@id/tv_title_track"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            android:layout_marginStart="@dimen/dp_8"
+            android:background="@color/color_gray"
+            android:layout_width="@dimen/dp_60"
+            android:layout_height="@dimen/dp_60" />
+
+        <TextView
+            android:layout_marginLeft="@dimen/dp_16"
```

Comment:
WARNING: [RtlHardcoded] Consider replacing `android:layout_marginLeft` with `android:layout_marginStart="@dimen/dp_16"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. **NOTE**: If your `minSdkVersion` is less than 17, you should add **both** the older left/right attributes **as well as** the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)